src/main/java/frc/trigon/robot/subsystems/MotorSubsystem.java

```diff
@@ -40,6 +43,18 @@ public MotorSubsystem() {
         REGISTERED_SUBSYSTEMS.add(this);
     }
 
+    /**
+     * Runs periodically. Only updated the mechanism if the robot is in replay mode or extensive logging is enabled.
+     */
```

Comment:
```suggestion
    /**
     * Runs periodically, to update the subsystem, and update the mechanism of the subsystem (if there is one).
     * This only updates the mechanism if the robot is in replay mode or extensive logging is enabled.
     * This function cannot be overridden, use {@linkplain MotorSubsystem#updatePeriodically} or {@linkplain MotorSubsystem#updateMechanism} (depending on the usage) instead.
     */
```
Make this function final so it could not be overridden.